deploy/ui/form.json

```diff
@@ -558,8 +558,7 @@
                   "label": "CPU request",
                   "type": "text",
                   "jsonPath": "$.spec.objects.console.resources.requests.cpu",
-                  "description": "Minimum processing power requested by the container",
-                  "default": "500m"
+                  "description": "Minimum processing power requested by the container"
```

Comment:
Do you mind to improve the description here and for others below telling what are the default value for each case if no value is provided?

Reply:
Done